zeroeq/subscriber.cpp

```diff
@@ -45,24 +45,27 @@ class Subscriber::Impl
         update();
     }
 
-    Impl(const URI& uri)
+    Impl(const URIs& uris)
         : _context(detail::getContext())
         , _browser(PUBLISHER_SERVICE)
         , _selfInstance(detail::Sender::getUUID())
     {
-        if (uri.getScheme() == DEFAULT_SCHEMA &&
-            (uri.getHost().empty() || uri.getPort() == 0))
+        for (const URI& uri : uris)
         {
-            ZEROEQTHROW(std::runtime_error(
-                std::string("Non-fully qualified URI used for subscriber")));
-        }
+            if (uri.getScheme() == DEFAULT_SCHEMA &&
+                (uri.getHost().empty() || uri.getPort() == 0))
```

Comment:
this could read more smoothly with a helper function:
```
if (!isFullyQualified(uri))
{ throw }
```
Or replacing the whole block:
`checkFullyQualified(uri);`